src/components/CategoryListItem.js

```diff
@@ -50,6 +50,7 @@ export class CategoryListItem extends React.PureComponent {
         }
         delayPressIn={0}
         Component={Touchable}
+        accessibilityLabel={`${title} (Anzahl verfügbarer Einträge ${count}) (Taste)`}
```

Comment:
nice additional info about the entries!
plese try to add a ":" to make voiceover and talback read the number better after the text?
`Anzahl verfügbarer Einträge: ${count}`